### DISCUSSION TITLE
Please provide API documentation for FastAPI itself

### DISCUSSION THREAD

**Discussion by mjog** [fastapi/fastapi#9079] [Questions]
### First check

* [ x] I added a very descriptive title to this issue.
* [ x] I used the GitHub search to find a similar issue and didn't find it.
* [ x] I searched the FastAPI documentation, with the integrated search.
* [ ] I already searched in Google "How to X in FastAPI" and didn't find any information.
* [ ] I already read and followed all the tutorial in the docs and didn't find an answer.
* [ ] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
* [ ] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
* [ ] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).
* [ x] After submitting this, I commit to:
    * Read open issues with questions until I find 2 issues where I can help someone and add a comment to help there.
    * Or, I already hit the "watch" button in this repository to receive notifications and I commit to help at least 2 people that ask questions in the future.
    * Implement a Pull Request for a confirmed bug.

The current docs are nice as a tutorial, but are not complete (as in, does not list/document all modules, classes and properties) and hence not helpful after moving past the beginner stage.

**Comment by jorgerpo** [comment]
If you use certain IDEs , I use VS Code, you can click on classes an functions to see the internals of the code.

**Comment by mjog** [comment]
> If you use certain IDEs , I use VS Code, you can click on classes an functions to see the internals of the code.

Sure, but that's not API documentation, which is what I am looking for.

**Comment by YuriiMotov** [comment]
Done: https://fastapi.tiangolo.com/reference/
